docs/introduction/faq-and-common-issues.md

```diff
@@ -93,3 +94,9 @@ Do not forget to set up a cron on your server to execute [`php phing cron`](/doc
 ## Why are you using entity data instead of entities for Symfony forms?
 We are using [entity data objects](entities.md#entity-data) instead of [entities](basics-about-model-architecture.md#entity)
 because Symfony forms need setters for all fields and we don't want to mess entities with them.
+
+## What is the configuration file services_test.yml good for?
```

Comment:
It seems to me like test are using only `services_test.yml`, could you somehow express that tests uses both configuration, but `services_test.yml` overrides `services.yml`?

Reply:
I tried, better? :slightly_smiling_face: